OpenSim/Simulation/Model/Model.cpp

```diff
@@ -1686,7 +1686,7 @@ void Model::writeMarkerFile(const string& aFileName)
  * @param aMarkerSet The new marker set to copy.
  * @return Number of markers that were successfully added to the model.
  */
-int Model::replaceMarkerSet(const SimTK::State& s, const MarkerSet& aMarkerSet)
+int Model::replaceMarkerSet(SimTK::State& s, const MarkerSet& aMarkerSet)
```

Comment:
Thanks for the review @aseth. The workflow of the GUI is different from invoking a setup file and executing it end to end because users expect to see the effects of changing markerSets immediately (e.g. in computing measurements or other places in the GUI). So the GUI expects to have a valid model with a consistent state after replacing the markerSet. If that's not recommended inside the method, I'm happy to make the call from OpenSimContext after the update/replace call, however we need to unify these two methods (update/replace) to make sure the API and GUI are executing the same workflow.